config.yml

```diff
@@ -1508,6 +1508,11 @@ nodes:
           __ENCODING__
           ^^^^^^^^^^^^
   - name: SourceFileNode
+    child_nodes:
+      - name: file_keyword
```

Comment:
I don't think we should add a token here, since it's going to always be the same type and have the same location as the rest of the node.